src/moves.rs

```diff
@@ -119,5 +119,28 @@ pub fn compute_move(game: &GameRequest) -> String {
             }
         }
     }
+
+    //  If the head of a Battlesnake is within the Hazard Sauce,
+    // its health points will more rapidly decline, loosing 15 health each turn.
```

Comment:
losing not loosing

Reply:
![image](https://user-images.githubusercontent.com/4238030/124369866-84430580-dc3e-11eb-825b-dde39eab9520.png)

Reply:
just because they typed it wrong doesn't mean that you have an excuse to leave it wrong

Reply:
It's unethical to misquote somebody and I have journalistic integrity

Reply:
I'm not going to approve until you fix the typo

Reply:
```suggestion
    // its health points will more rapidly decline, losing 15 health each turn.
```

Reply:
if i commit this can we even get this merged